frontend/src/app/commonComponents/MultiplexResultsGuidance.tsx

```diff
@@ -1,132 +0,0 @@
-import React from "react";
```

Comment:
⚠️ this was deleted because the logic in here was separated out into 3 new files
- `FluResultGuidance.tsx`
- `RsvResultGuidance.tsx`
- `ResultsGuidance.tsx`